code/API_definitions/quality-on-demand.yaml

```diff
@@ -197,24 +225,6 @@ paths:
                     status: 400
                     code: INVALID_ARGUMENT
                     message: "Expected property is missing: device"
-                InsufficientDeviceProperties:
-                  summary: Device must be identified by at least one parameter
-                  value:
-                    status: 400
-                    code: INVALID_ARGUMENT
-                    message: "Insufficient properties specified: device"
-                InconsistentDeviceProperties:
-                  summary: Device parameters provided identify different devices
-                  value:
-                    status: 400
-                    code: INVALID_ARGUMENT
-                    message: "Multiple inconsistent parameters specified: device"
-                CannotIdentifyDevice:
-                  summary: No device can be identified from provided parameters
-                  value:
-                    status: 400
-                    code: INVALID_ARGUMENT
-                    message: "Unable to identify device from specified parameters: device"
                 InvalidDevicePublicPortValue:
```

Comment:
In general I think we may rely on the generic 400 for errors when input does not comply with schema: mandatory property missing, property values out of range, etc

Reply:
Yes, the DurationOutOfRangeForQoSProfile case is relevant and not covered by the compliance to schema. In these cases  we can design an API dedicated code, e.g.  QUALITY_ON_DEMAND.DURATION_OUT_OF_RANGE_FOR_QOS_PROFILE. Or something shorter, but API_NAME.ERROR_CODE is the expected format.

Reply:
Will do so. Just a question if understood it correctly:

Looking on CAMARA_common, the template is:

```
            GENERIC_400_{{SPECIFIC_CODE}}:
              description: Specific Syntax Exception regarding a field that is relevant in the context of the API
              value:
                status: 400
                code: { { SPECIFIC_CODE } }
                message: { { SPECIFIC_CODE_MESSAGE } }
```

merging this with 

```
            DurationOutOfRangeForQoSProfile:
              summary: The requested duration is out of the allowed range for the specific QoS profile
              value:
                status: 400
                code: OUT_OF_RANGE
                message: "The requested duration is out of the allowed range for the specific QoS profile"
```

my proposal would be ("GENERIC_400_{{SPECIFIC_CODE}}" does not make sense to me, and I haven't found it anywhere):

```
            DurationOutOfRangeForQoSProfile:
              description: The requested duration is out of the allowed range for the specific QoS profile
              value:
                status: 400
                code: QUALITY_ON_DEMAND.DURATION_OUT_OF_RANGE
                message: The requested duration is out of the allowed range for the specific QoS profile
```

ok?

Reply:
QUALITY_ON_DEMAND.DURATION_OUT_OF_RANGE introduced in https://github.com/camaraproject/QualityOnDemand/pull/326/commits/5c9bddea1a3487d98bd59f89c7492ea45bf0c4c6